App.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, FlatList, Dimensions } from 'react-native';
```

Comment:
Always remember to revert `App.js` to the `master` branch's version - if you don't, you could introduce conflicts.